vud.theme.inc

```diff
@@ -72,16 +72,9 @@ function vud_widget_get_names() {
  * Implementation of hook_theme().
  */
 function vud_theme() {
-  return array(
-    'vud_widget' => array(
-      'function'  => 'vud_widget_proxy',
-      'variables' => array('entity_id' => NULL, 'type' => NULL, 'tag' => NULL, 'widget_theme' => NULL, 'readonly' => NULL, 'widget_message_code' => NULL),
-    ),
-    'vud_votes' => array(
-      'function'  => 'vud_votes_proxy',
-      'variables' => array('entity_id' => NULL, 'type' => NULL, 'tag' => NULL, 'widget_theme' => NULL),
-    ),
-  );
+  return [
```

Comment:
You are removing hook_theme entries, any reason? I see you are returning renderable arrays on widgets' `build()`, so where is the related entry here?